Maitreyee/addlatlon

maitrejoshi opened the pull request:
added latitude and longitude to display on Veritas

maitrejoshi committed:
added lat/lon

maitrejoshi committed:
change name to geocoordinates

maitrejoshi committed:
Changed reader files to geocoordinates

maitrejoshi committed:
changed command argument to Geographical Coordinat

arjunskND committed:
Getting rid of un needed changes

arjunskND committed:
Reverted to GEO_COORDINATES. GEOGRAPHICAL_COORDINATES will require wholesale changes

bcasselsND commented on PotreeConverter/src/main.cpp:
```diff
@@ -85,7 +85,7 @@ PotreeArguments parseArguments(int argc, char **argv){
 	args.addArgument("intensity-range", "");
 	args.addArgument("output-format", "Output format can be BINARY, LAS or LAZ. Default is BINARY");
-	args.addArgument("output-attributes,a", "can be any combination of RGB, INTENSITY, CLASSIFICATION, GPS_TIME, RTK_POSE, RTK_ORIENT, and DUAL_PLUS_CONFIDENCE. Default is RGB.");
+	args.addArgument("output-attributes,a", "can be any combination of RGB, INTENSITY, CLASSIFICATION, GPS_TIME, RTK_POSE, RTK_ORIENT, DUAL_PLUS_CONFIDENCE AND GEO_COORDINATES. Default is RGB.");
```
`DUAL_PLUS_CONFIDENCE AND GEO_COORDINATES` => `DUAL_PLUS_CONFIDENCE, and GEO_COORDINATES`

bcasselsND reviewed (approved):
Nit.

maitrejoshi committed:
changes geoCoords to Lat Lon

arjunskND committed:
Some formattting

arjunskND committed:
Dynamically calculating latlon:

- Had to make CMakeList changes to use VTM code in DataSchemas
- Using metadata processing jsoono file to use that code
- Best to use invalid field and checking of file exists?
- Grouped lat long attributes under geographical_coorodinates, proobably also include altitude

maitrejoshi committed:
added altitude to fbpoints and classified points

arjunskND committed:
Tab vs. space

arjunskND committed:
Some formatting
